debug/testlib.py

```diff
@@ -748,16 +750,15 @@ def classSetup(self):
         BaseTest.classSetup(self)
 
         if gdb_cmd:
-            self.gdb = Gdb(self.server.gdb_ports, gdb_cmd, binary=self.binary)
+            self.gdb = Gdb(self.server.gdb_ports, gdb_cmd, timeout = self.target.timeout_sec,  binary=self.binary)
         else:
-            self.gdb = Gdb(self.server.gdb_ports, binary=self.binary)
+            self.gdb = Gdb(self.server.gdb_ports, timeout = self.target.timeout_sec, binary=self.binary)
```

Comment:
This will probably fail the linter (I *think*; I haven't used pylint myself, but I assume it enforces PEP 8). Python's official style guide is to not put any spaces around the equals sign in named arguments. Also only one space after commas is allowed.